src/components/Rocket.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { reserveRocket } from '../redux/rockets/rocketsSlice';
+
+const Rocket = ({
+  id, name, description, img, reserved,
+}) => {
+  const dispatch = useDispatch();
+
+  return (
+    <div className="flex justify-start items-center h-44" id={id} data-testid="main_container">
+      <img className="w-60 h-full" src={img} alt={name} />
+      <div className="flex flex-col justify-between px-5 h-full">
+        <h2 className="font-semibold text-xl">{name}</h2>
+        <p>
+          {reserved && <span className="p-0.5 mr-2 rounded-md text-white bg-cyan-600">Reserved</span>}
+          {description}
+        </p>
+        {!reserved && (
+        <button
+          className="w-32 p-2 mr-2 rounded-md text-white bg-blue-500 hover:bg-blue-600"
+          type="button"
+          onClick={() => {
+            dispatch(reserveRocket(id));
+          }}
+        >
+          Reserve Rocket
```

Comment:
- [ ] Kindly make sure this button has the same design as in the template image provided. 

Your button  | The button in the image design
--|---
<img width="392" alt="image" src="https://user-images.githubusercontent.com/65068771/224190504-fda596a5-172c-4710-87e0-5c883230c666.png"> | <img width="392" alt="image" src="https://user-images.githubusercontent.com/65068771/224190686-20049eb2-c938-4d69-9949-896ec76fee97.png">

Reply:
This issue does not appear when I run the project in my computer but I increased the button's size anyways, thank you so much👌, @andersonlebon

Reply:
hey @jmonto55  maybe your screen was zoomed in